### PR TITLE
ci: Add julia and formatter on macOS and Linux 

### PR DESCRIPTION
**Pull request by jayvdb**
Sits on top of https://github.com/adsharma/py2many/pull/60

### CODE REVIEW

File: py2many/cli.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def rust_settings(args):
 
 
 def julia_settings(args):
-    return LanguageSettings(JuliaTranspiler(), ".jl", ["jlfmt"])
+    return LanguageSettings(JuliaTranspiler(), ".jl", ["format.jl"])
```
Comment (adsharma) [inline py2many/cli.py:72]: This is ok as long as you prefix it with ijulia:

```
$ time -p bin/format.jl ~/src/py2many/tests/expected/binit.jl 
real 17.34
user 17.61
sys 0.70
$ time -p ijulia bin/format.jl ~/src/py2many/tests/expected/binit.jl 
real 1.98
user 2.19
sys 0.72
```

Comment (jayvdb) [reply py2many/cli.py:72]: Adding a dependency on ijulia is very expensive on CI, and more daunting for newcomers wanting to figure out how to install get pyjl working.

It is easier for you to create your own `format.jl` in your PATH which has your preferred logic.

Comment (adsharma) [reply py2many/cli.py:72]: ijulia is not a new package. It's a one line shell script:

```
julia -O0 --compile=min --startup=no $*
```

I tried adding a similar shebang line to format.jl, but it didn't reduce the runtime.

Making CI less expensive in overall runtime is why I'm interested in this.

Comment (adsharma) [reply py2many/cli.py:72]: Namespace collision I wasn't aware of. The change is so trivial that it doesn't seem to be worth the complexity of adding a new script.

How about:

```
return LanguageSettings(JuliaTranspiler(), ".jl", ["julia", "-O0", "--compile=min", "--startup=no", "format.jl"])
```

I pasted performance numbers on my machine above. Without these args, formatter is 10x slower on my machine.

Comment (jayvdb) [reply py2many/cli.py]: done

**Pull request merged**